### ISSUE TITLE
Missing measurement units might break down keyframe animations

### ISSUE THREAD

**Issue by recon** [fmarcia/UglifyCSS#18]
I've recently encountered a problem with some keyframe animations. Supposedly we would have this piece of code: 

``` css
@-webkit-keyframes bounce {
    0%, 100% { 
        -webkit-transform: scale(0.0) 
    }
    50% { 
        -webkit-transform: scale(1.0) 
    }
}
@keyframes bounce {
    0%, 100% {
        transform: scale(0.0);
        -webkit-transform: scale(0.0);
    } 
    50% {
        transform: scale(1.0);
        -webkit-transform: scale(1.0);
    }
}
```

Ok, after it went through UglifyCSS, the result would be (the beautification is done by me - it's not relevant):

``` css
@-webkit-keyframes bounce {
    0,100% {
        -webkit-transform: scale(0.0);
    }
    50% {
        -webkit-transform: scale(1.0);
    }
}

@keyframes bounce {
    0,100% {
        transform: scale(0.0);
        -webkit-transform: scale(0.0);
    }  
    50% {
        transform: scale(1.0);
        -webkit-transform: scale(1.0);
    }
}
```

Note this `0%` becoming `0`. The missing percent sign seems to break the animation - apparently `0%` != `0` when it comes to keyframe animations, and this also applies to `0s`. 
Basically, in this scenario those `0%...100%` are acting like selectors (I hope this is the proper term here), rather than values, and they should be kept intact.

Here is a related article with more info: http://www.sitepoint.com/avoid-css3-animation-minification-muddles/

Thank you!
